streams/src/test/java/org/apache/kafka/streams/tests/SmokeTestDriver.java

```diff
@@ -232,7 +236,7 @@ public static void verify(String kafka, Map<String, Set<Integer>> allData, int m
         }
         int retry = 0;
         final long start = System.currentTimeMillis();
-        while (System.currentTimeMillis() - start < TimeUnit.MINUTES.toMillis(3)) {
+        while (System.currentTimeMillis() - start < TimeUnit.MINUTES.toMillis(6)) {
```

Comment:
Not sure what's happening here. But `6` does still not seem to be large. Why change it in the first place?